spec/factories/user_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    name 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.